ddsrouter_yaml/test/unittest/entities/topic/YamlGetEntityTopicTest.cpp

```diff
@@ -181,6 +197,24 @@ TEST(YamlGetEntityTopicTest, get_real_topic)
 
         ASSERT_THROW(YamlReader::get<core::types::RealTopic>(yml, "topic", LATEST), utils::ConfigurationException);
     }
+
+    // Topic reliable
```

Comment:
Update the test comment and move this with the positive cases

Reply:
Extended

Reply:
I meant to add the comment in the test header :laughing: 
But it does not matter